auditlog/models.py

```diff
@@ -26,13 +26,15 @@ class LogEntryManager(models.Manager):
     Custom manager for the :py:class:`LogEntry` model.
     """
 
-    def log_create(self, instance, **kwargs):
+    def log_create(self, instance, force_log: bool = False, **kwargs):
```

Comment:
@hramezani This is a recent addition. When the receivers were calling this method, the `changes` kwarg would equal `"null"` when changes were `None`. This transformation was caused by the `json.dumps` method being called on `None`. 

Because we don't need to transform with the json library anymore, we need a way to signal that a log should be created when there are no changes. Specifically this is required for the new `ACCESS` action type. I extended the idea of passing  along the `force_log` flag in those cases.